e2e_playwright/st_audio_input_test.py

```diff
@@ -136,6 +137,27 @@ def test_audio_input_label_visibility_snapshot(
     )
 
 
+@pytest.mark.only_browser("chromium")
+def test_audio_input_file_download(app: Page):
+    """Test that the audio input file can be downloaded."""
+    app.context.grant_permissions(["microphone"])
+
+    audio_input = app.get_by_test_id("stAudioInput").nth(1)
+    audio_input.get_by_role("button", name="Record").click()
+    app.wait_for_timeout(1500)
```

Comment:
nit: this is probably intended to record for 1.5seconds? Maybe add a small comment here why the timeout is requried.